src/Kentico.Xperience.Algolia/Indexing/AlgoliaIndexStore.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Kentico.Xperience.Algolia.Admin;
+
+namespace Kentico.Xperience.Algolia.Indexing
```

Comment:
```suggestion
namespace Kentico.Xperience.Algolia.Indexing;
```

Use file-scoped namespaces.

Reply:
changed everywhere